hw1/01-selectors.md

```diff
@@ -2,6 +2,33 @@
 
 1. plate
 2. bento
-3.
-4.
-5. ...
+3. #fancy
+4. plate apple
+5. #fancy pickle
+6. apple.small
+7. orange.small
+8. bento orange.small
+9. plate, bento
+10. \*
```

Comment:
Since vscode is not allowing me to use *, so I added a escape character before the *